ci/infra/testrunner/README.md

```diff
@@ -234,7 +232,7 @@ Be sure you don't specify the `terraform.tfdir` directory, so terraform configur
 You use your `id_rsa` keys to connect to the cluster nodes, as the `shared_id` is not available.
 
 ```
-ssh_option: "id_rsa"
+ssh_key: "path/to/id_rsa"
```

Comment:
I missed this earlier, but it's not a huge deal - typically you'd want to start this with a `/` to indicate that it needs to be an absolute path.  But that can be fixed later.
```suggestion
ssh_key: "/path/to/id_rsa"
```